noxfile.py

```diff
@@ -13,7 +13,7 @@
 
 @nox.session(reuse_venv=True, python=python)
 def tests(session):
-    session.install(".", "pytest", "pytest-cov")
+    session.install(".", "pytest", "pytest-cov", "flask-socketio>4.2.0")
```

Comment:
`"flask-socketio` should already be included (if necessary) with the `.` target

Reply:
I will check again the problem of not having the > 4.2 it was that it was installing the version 2.9.3 (I think). And I needed flask-socketio bigger than 3.3 because of test_backend_mpi.py. I introduced a test that to emulate a debug session for the server and I had to connect  the HTTP cookies sesstion to the flask-io socket. This feature was introduced in flask-socketio 3.3

Reply:
If the version is upgraded (and everything still works) that is fine, but it should be done in setup.py